include/swift/AST/DiagnosticsSema.def

```diff
@@ -2482,6 +2482,9 @@ ERROR(tuple_pattern_in_non_tuple_context,none,
 ERROR(closure_argument_list_tuple,none,
       "contextual closure type %0 expects %1 argument%s1, "
       "but %2 were used in closure body", (Type, unsigned, unsigned))
+ERROR(closure_argument_list_single_tuple,none,
```

Comment:
I tried to use `%switch` but couldn't find it in the code base, therefore I use `%select` instead.